flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64", "aarch64"]
```

Comment:
you don't need this, these are the only to architecture supported om 20.08

Reply:
I use some python packages, which are only available for this architectures. This is to avoid problems, when other architectures are added.

Reply:
again, on 20.08 these are the only architecture supported.